CommandsAndAdaptors/parser_adaptor.py

```diff
@@ -1,6 +1,6 @@
 class ParsedInput:
```

Comment:
Rename it by parserAdapter 
And it should inherit from IparserAdapter

Reply:
Shouldn't it inherit from InputAdapter?

Reply:
Yes it should